patches/vkd3d-proton/build-number-no-git.patch

```diff
@@ -1,12 +1,18 @@
 diff --git a/meson.build b/meson.build
-index 2eda9c64..06113bc1 100644
+index 2eda9c64..9c07940c 100644
 --- a/meson.build
 +++ b/meson.build
-@@ -101,6 +101,7 @@ endif
+@@ -101,11 +101,13 @@ endif
  
  vkd3d_build = vcs_tag(
    command : ['git', 'describe', '--always', '--exclude=*', '--abbrev=15', '--dirty=0'],
-+  fallback: run_command('sh', '-c', 'printf "%x" $(date +%s)').stdout(),
++  fallback: run_command('sh', '-c', 'cat', files('BUILD')).stdout().strip(),
```

Comment:
I guess this won't work, `sh -c` expects command as one argument. But you probably don't need shell wrap here at all, so try `run_command('cat', files('BUILD'))` instead.